docs/source/data_ingestion_and_preparation/data_preparation_concepts.md

```diff
@@ -25,24 +25,24 @@ docset = context.read\
 
-* **doc_id:** A unique identifier for the document. Defaults to a uuid.
+* **doc_id:** A unique identifier for the document. Defaults to a UUID.
 
-* **parent_id:** In Sycamore, certain operations create parent-child relationships between documents. For example, the explode transform promotes elements to be top-level documents, and these documents retain a pointer to the document from which they were created using the parent_id field. For those documents which have no parent, parent_id is None.
+* **parent_id:** In Sycamore, certain operations create parent-child relationships between Documents. For example, the explode transform promotes elements to be top-level documents, and these documents retain a pointer to the document from which they were created using the parent_id field. For those Documents which have no parent, parent_id is None.
 
-* **binary_representation, text_representation:** The raw content of the document in stored in the appropriate format. For example, the content of a PDF document will be stored as the binary_representation, while an HTML page will be stored as text_representation.
+* **binary_representation, text_representation:** The raw content of the Document in stored in the appropriate format. For example, the content of a PDF document will be stored as the binary_representation, while an HTML page will be stored as text_representation.
 
-* **elements:** a list of elements belonging to this document. A document does not necessarily always have elements, for instance, before a document is chunked.
+* **elements:** A list of elements belonging to this Document. A Document does not necessarily always have elements. For instance, before a Document is chunked it will not have any elements.
 
-* **properties:** A collection of system or customer defined properties, for instance, a PDF document might have title and author properties.
+* **properties:** A collection of system or customer defined properties. For instance, a PDF document might have title and author properties.
 
 ### Element
 
-It is often useful to process different parts of a document separately. For example, you might want to process tables differently than text paragraphs, and typically small chunks of text are embedded separately for vector search. In Sycamore, these chunks are called elements. Like documents, elements contain a text or binary representations and collection of properties that can be set by the user or by built-in transforms.
+It is often useful to process different parts of a Document separately. For example, you might want to process tables differently than text paragraphs, and typically small chunks of text are embedded separately for vector search. In Sycamore, these chunks are called Elements. Like Documents, Elements contain a text or binary representations and collection of properties that can be set by the user or by built-in transforms.
```

Comment:
contain a text or binary *representation*

(or leave in the s and remove the 'a')

Reply:
Ack